docs/hugo/content/tutorials/upgrade-resource-version/_index.md

```diff
@@ -0,0 +1,677 @@
+---
+title: "Upgrading a resource"
+type: docs
+description: "Step by step walkthrough of upgrading a resource version"
+---
+
+While adding a new version of an existing resource to Azure Service Operator (ASO) is _relatively_ straightforward, it's important to remember that this is relative to hand writing the resource from scratch. 
+
+Our code generator does much of the heavy lifting, but the process isn't trivial and requires a good understanding of the Azure Resource Manager (ARM) API, the ASO code generator, and how the operator works.
+
+This walkthrough will guide you through the process of upgrading a resource version in ASO. We'll be upgrading the `DatabaseAccount` resource for the `Microsoft.DocumentDB` provider, but the process is similar for any resource.
+
+## Identify the version to upgrade
+
+We're going to be importing a specific version of the resource - so we need to work out which version to pick.
+
+### Check the Azure ARM documentation
+
+Find the documentation page for the resource version you want to upgrade and check which versions are available.
+
+Staring with a [web search](https://www.bing.com/search?q=azure+arm+bicep+Microsoft+documentdb), we find the latest stable version is `2023-11-15`. It is usually a good idea to upgrade to the latest **stable** version, unless you have a specific preview feature in mind.
```

Comment:
Should [microsoft-documentdb-database-accounts.png](https://github.com/Azure/azure-service-operator/pull/3967/files?short_path=c8b1b3a#diff-f26cda580cae95ef017549c0d910db15a6295b68e7d97fe644d16f7e796bfacf) be specified in this section?